test/tests/groovy/bigIntsTest.js

```diff
@@ -0,0 +1,155 @@
+/*
+ *  Copyright 2017 TWO SIGMA OPEN SOURCE, LLC
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+var BeakerXPageObject = require('../beakerx.po.js');
+var beakerxPO;
+
+describe('big ints tests', function () {
+
+  beforeAll(function () {
+    beakerxPO = new BeakerXPageObject();
+    beakerxPO.runNotebookByUrl('/notebooks/test/notebooks/groovy/BigIntsTest.ipynb');
+  }, 2);
+
+  afterAll(function () {
+    beakerxPO.closeAndHaltNotebook();
+  });
+
+  describe('64-bit Longs labels and values', function () {
+
+    it('Table has proper column labels', function () {
+     var tableColumnLabel;
+     var cellText;
+     var codeCell = beakerxPO.runCodeCellByIndex(0);
+     beakerxPO.kernelIdleIcon.waitForEnabled();
+
+     tableColumnLabel = beakerxPO.getTableColumnLabel(0, 0)
+     cellText = tableColumnLabel.getText()
+     expect(cellText).toMatch('time')
+
+     tableColumnLabel = beakerxPO.getTableColumnLabel(0, 1)
+     cellText = tableColumnLabel.getText()
+     expect(cellText).toMatch('next_time')
+
+     tableColumnLabel = beakerxPO.getTableColumnLabel(0, 2)
+     cellText = tableColumnLabel.getText()
+     expect(cellText).toMatch('temp')
+     });
+
+    it('Table has proper cell values', function () {
+     var tableCell
+     var cellText
+     tableCell = beakerxPO.getTableCell(0, 0, 1)
+     cellText = tableCell.getText()
+     expect(cellText).toMatch('1912276048007')
```

Comment:
what are these values? when i run the notebook the values i get are:
```
1516039311030000007	1516039311030000077	14.600
1516039311030000014	3032078622060000176	18.100
1516039311030000021	4548117933090000297	23.600
```

Reply:
Try to run notebook from tests/notebooks/groovy folder. Notebook there is a bit different from the original one and 'milis' is changed from date function to a constant, as Vladimir suggested.